cmd/aws-iam-authenticator/server.go

```diff
@@ -139,12 +123,16 @@ func init() {
 		"master is the URL to the api server")
 	viper.BindPFlag("server.master", serverCmd.Flags().Lookup("master"))
 
-	serverCmd.Flags().String(
-		"address",
+	serverCmd.Flags().String("address",
 		"127.0.0.1",
 		"IP Address to bind the server to listen to. (should be a 127.0.0.1 or 0.0.0.0)")
 	viper.BindPFlag("server.address", serverCmd.Flags().Lookup("address"))
 
+	serverCmd.Flags().StringSlice("backend-mode",
+		[]string{mapper.ModeCRD, mapper.ModeConfigMap, mapper.ModeFile},
+		fmt.Sprintf("Ordered list of backends to get mappings from. Comma-delimited list of: %s", strings.Join(mapper.BackendModeChoices, ",")))
```

Comment:
Can you specify what "ordered" means in this context? First one that matches wins? Last one wins?

Reply:
first one wins, added clarification.